src/main/java/com/contrastsecurity/http/SecurityCheckForm.java

```diff
@@ -52,12 +49,31 @@
     @SerializedName("origin")
     private String origin = "OTHER";
 
-    public SecurityCheckForm(String applicationId) {
+    /**
+     * The job start time of the jenkins job
+     * @param jobStartTime New value for job start time.
+     * @return the start time of the jenkins job
+     */
+    @SerializedName("job_start_time")
+    private Long jobStartTime;
+
+    public static SecurityCheckForm securityCheckForm(String applicationId) {
```

Comment:
public SecurityCheckForm(String applicationId)
public SecurityCheckForm(String applicationId, AgentType agentLanguage)

Add a set method for jobStartTime